docs/attachment_job.md

```diff
@@ -0,0 +1,24 @@
+# Doc for Basic form of an Attachment Job
+
+Work generator creates an API call for comments. 
+
+An example of an API call to comments is:
+- https://api.regulations.gov/v4/comments/CMS-2022-0012-0392/?api_key=<API_KEY>
+
+The returned json object may or may not contain an attachment.
```

Comment:
Could you provide example jsons? Or at least what the section looks like with/without an attachment.

Reply:
@AbdullahAlramyan brought up a good point about linking comments to attachments.  We will need to have the `id` of the comment in the jobs as well.  This will allow us to pass this information to the client and then back to the work server so it can store the data appropriately.